setup.py

```diff
@@ -426,36 +426,7 @@ def run(self):
 setup(
     name="transformers",
     version="4.28.0.dev0",  # expected format is one of x.y.z.dev0, or x.y.z.rc1 or x.y.z (no to dashes, yes to dots)
-    author="The Hugging Face team (past and future) with the help of all our contributors (https://github.com/huggingface/transformers/graphs/contributors)",
-    author_email="transformers@huggingface.co",
-    description="State-of-the-art Machine Learning for JAX, PyTorch and TensorFlow",
-    long_description=open("README.md", "r", encoding="utf-8").read(),
-    long_description_content_type="text/markdown",
```

Comment:
We are losing those apparently.

Reply:
It goes to `project.readme` in `pyproject.toml`. The content type will be inferred from the extension (`.md` or `.rst`).